ui/src/app/App.tsx

```diff
@@ -127,7 +128,11 @@ export const App = (): JSX.Element => {
       </Section>
     );
   } else if (connected) {
-    content = <Routes />;
+    content = (
+      <FileContext.Provider value={fileContextStore}>
```

Comment:
Curious why context and not redux?

Reply:
All the websocket data we get goes into the redux store, we don't have any async way to get responses directly into a component. With these large files it didn't feel like it should be stored in the redux store so this context provides a way to get the file data to the component.

Does this seem reasonable do you think?

Reply:
I'm going to land this to unblock me, but if you have thoughts about if this could be improved then let me know.